includes/admin/settings/class-settings-display.php

```diff
@@ -118,9 +118,9 @@ public function get_settings() {
 							'wrapper_class' => 'give-hidden give-title-prefixes-settings-wrap',
 							'style'         => 'width: 30%',
 							'options'       => array(
-								'Mr'  => __( 'Mr', 'give' ),
-								'Mrs' => __( 'Mrs', 'give' ),
-								'Ms'  => __( 'Ms', 'give' ),
+								'Mr.'  => __( 'Mr.', 'give' ),
```

Comment:
@mehul0810 create a helper function with a filter for default title fixes and use this function here.